### PR TITLE
All GCM related things removed.

### PR DESCRIPTION
**Pull request by tasomaniac**
GCM is not working and we have really old GCM implementation in our app. 

If we want to enable it in the future, we can re-write it.

### CODE REVIEW

File: app/src/main/java/org/gdg/frisbee/android/fragment/SettingsFragment.java [hunk 1]
```diff
@@ -74,7 +63,7 @@
         public boolean onPreferenceChange(Preference preference, Object o) {
             final String homeGdg = (String) o;
 
-            if (mGoogleApiClient.isConnected() && PrefUtils.isGcmEnabled(getActivity())) {
+            if (mGoogleApiClient.isConnected()) {
```
Comment (friedger) [inline app/src/main/java/org/gdg/frisbee/android/fragment/SettingsFragment.java]: :+1:

File: app/src/main/java/org/gdg/frisbee/android/utils/PrefUtils.java [hunk 2]
```diff
@@ -197,16 +161,6 @@ public static void setAchievementSteps(@NonNull final Context context,
             .apply();
     }
 
-    public static boolean isAppStateMigrationSuccessful(final Context context) {
```
Comment (friedger) [inline app/src/main/java/org/gdg/frisbee/android/utils/PrefUtils.java]: not gcm related

**Comment by friedger** [comment]
I think we can still reuse most of the code for the new implemenation. However, the get the methods count down, LGTM

**Comment by tasomaniac** [comment]
We know the code is here. It's been unused for years. Thank you for merging
:)

**Pull request merged**